libc/src/__support/CPP/new.cpp

```diff
@@ -7,7 +7,6 @@
 //===----------------------------------------------------------------------===//
 
 #include "new.h"
-#include <stdlib.h>
```

Comment:
this include shouldn't be removed since it's being used directly in this file, even though it could be used indirectly through `new.h`.

Reply:
ok, thanks